build/test.sh

```diff
@@ -28,5 +28,5 @@ if [ -z "${PKG}" ]; then
   exit 1
 fi
 
-go test -v \
+go test -p 1 -v \
```

Comment:
Should we really disable parallel executions here?

Reply:
Seems not needed any more.